Bottom: 0ec06489eb21df1d65de2a3bf5e57001f93c85ac
Top:    0d6e0aea6c69f726c0e8c14c25866cd0a3a02549
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-05 01:58:01 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
